payroll_contract_advantages/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    "name": "Payroll Contract Advantages",
+    "version": "15.0.3.0.0",
```

Comment:
Whenever you migrate to a new version you should reset the semantic versioning to 1.0.0. So this should be 15.0.1.0.0.

Reply:
Okay will do today.